BookingSystem/Frontend/src/hotelSearch/containers/HotelSearchContainer.js

```diff
@@ -28,36 +26,45 @@ class HotelSearchContainer extends Component {
   state = {
     startDate: new Date(),
     endDate: new Date(),
-    roomSize: 1,
+    roomSize: { label: 1, value: 1 },
     city: "",
     country: ""
   };
 
   componentDidMount() {
-    this.props.getHotels({});
+    const data = this.getFormRequestData();
+    this.props.getHotels(data);
+    this.props.loadRoomSizeOptions();
   }
 
+  getFormRequestData = () => ({
+    startDate: this.state.startDate,
+    endDate: this.state.endDate,
+    roomSize: this.state.roomSize.value,
+    cityId: this.state.city.value,
+    countryId: this.state.country.value,
+    page: this.props.pageInfo.page,
+    pageSize: this.props.pageInfo.pageSize
+  });
+
   handleSubmit = event => {
     event.preventDefault();
-    const data = {
-      startDate: this.state.startDate,
-      endDate: this.state.endDate,
-      roomSize: this.state.roomSize,
-      cityId: this.state.city.cityId,
-      countryId: this.state.country.countryId
-    };
+    const data = this.getFormRequestData();
     this.props.getHotels(data);
   };
 
   handleReset = event => {
     this.setState({
```

Comment:
why don't you define default options separately to reuse them here and in the constructor instead of duplicating this code?